chart/templates/mayastor/obs/obs-callhome-deployment.yaml

```diff
@@ -54,7 +54,7 @@ spec:
             requests:
               cpu: {{ .Values.obs.callhome.resources.requests.cpu | quote }}
               memory: {{ .Values.obs.callhome.resources.requests.memory | quote }}
-        {{- if .Values.obs.callhome.enabled }}
+        {{- if .Values.eventing.enabled }}
```

Comment:
Shouldn't we have "ifdef" out the aggregator url above?

Reply:
Ok

Reply:
Done